crypto/armor_test.go

```diff
@@ -8,7 +8,9 @@ import (
 	"testing"
 
 	cmtcrypto "github.com/cometbft/cometbft/crypto"
+	"github.com/cometbft/cometbft/crypto/armor"
 	"github.com/cometbft/cometbft/crypto/xsalsa20symmetric"
```

Comment:
```suggestion
	"github.com/cosmos/cosmos-sdk/crypto/xsalsa20symmetric"
```

We shouldn't need cometbft's armor, and I just realized that we are using cometbft's xsalsa20symmetric while we have our own copy of this.